unitree_legged_msgs/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>unitree_legged_msgs</name>
```

Comment:
Let's name with a1_xxx, since other unitree patterns are not within consideration for now.